src/htdocs/css/_features.scss

```diff
@@ -46,9 +21,5 @@
 }
 
 .feature-image {
-  width: 75px;
-}
-
-.feature-item > a > .feature-title {
-  font-size: 1.1em;
+  width: 300px;
```

Comment:
This can be deleted, the width is always defined as 100%

```
.feature-main > a > .feature-image {
    background-repeat: no-repeat;
    background-size: 100%;
    padding-bottom: 65%;
    width: 100%;
}
```